test/new-e2e/system-probe/test/micro-vm-init.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eo xtrace
+
+GOVERSION=$1
+KITCHEN_DOCKERS=/kitchen-docker
+
+# Add provisioning steps here !
+## Set go version correctly
+eval $(gimme $GOVERSION)
+## Start docker
+systemctl start docker
+## Load docker images
+find $KITCHEN_DOCKERS -maxdepth 1 -type f -exec docker load -i {} \;
+
+# VM provisioning end !
+
+# Start tests
+IP=$(ip -f inet addr show $(ip route get $(getent ahosts google.com | awk '{print $1; exit}') | grep -Po '(?<=(dev ))(\S+)') | sed -En -e 's/.*inet ([0-9.]+).*/\1/p')
```

Comment:
this long command should be documented, and possibly broken up into steps so it is clear what it is doing.

Reply:
`$(getent ahosts google.com | awk '{print $1; exit}')` couldn't this be replaced with a known public IP like `8.8.8.8`?

Reply:
I found a simpler version of this command